packages/google_atlas/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: google_atlas
 description: Google Map Provider for Atlas, an extensible map abstraction for Flutter with support for multiple map providers
-version: 0.3.0
+version: 0.4.0
```

Could this be a minor release?
```suggestion
version: 0.3.1
```